edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/RestartPolicyManager.cs

```diff
@@ -71,8 +71,8 @@ bool ShouldRestart(IRuntimeModule module)
                 TimeSpan coolOffPeriod = this.GetCoolOffPeriod(module.RestartCount);
                 TimeSpan elapsedTime = DateTime.UtcNow - module.LastExitTimeUtc;
 
-                bool shouldRestart = elapsedTime > coolOffPeriod;
-                if(!shouldRestart)
+                bool shouldRestart = elapsedTime > TimeSpan.Zero ? elapsedTime > coolOffPeriod : true;
```

Comment:
Suggest to put a comment here to explain the scenario why elapsed time can be negative.

Reply:
done